src/operators.js

```diff
@@ -8,6 +8,7 @@ const unaryOperators = {
   typeof: true,
   void: true,
   delete: true,
+  await: true,
```

Comment:
It doesn't look like this is being used. If it does get used, is it correct? `await` is a contextual keyword like `yield` and `async`.